docs/apache-airflow/tutorial/pipeline.rst

```diff
@@ -170,7 +170,7 @@ Here we select completely unique records from the retrieved data, then we check
               FROM employees_temp
           ) t
           ON CONFLICT ("Serial Number") DO UPDATE
-          SET "Serial Number" = excluded."Serial Number";
+          SET "Serial Number" = t."Serial Number";
```

Comment:
Looks like the issue with the subquery alias was already handled in #28353. The update should be done with `excluded` values. 

However, this merge query doesn't make any functional sense. It's trying to insert into `employees`, if there is a unique constraint violation on `Serial Number` then update `Serial Number` with the value that caused the violation (aka update `Serial Number` with its own value). If there is a unique constraint violation, it would make more practical sense to update other columns from the `excluded` values.
```suggestion
          SET
                "Employee Markme" = excluded."Employee Markme",
                "Description" = excluded."Description",
                "Leave" = excluded."Leave";
```
WDYT?

Reply:
@josh-fell, thanks for the feedback! 

Yep, that definitely makes much more sense. "Serial Number" is a PK so it's the other fields should be updated on conflict.

Reply:
@tsoud would you like to complete this PR?

Reply:
> @tsoud would you like to complete this PR?

I had drifted away from this. Thanks @eladkal!